app/controllers/api/v1/orders_controller.rb

```diff
@@ -96,8 +96,23 @@ def destroy
         render json: {}
       end
 
+      def summary
```

Comment:
Do we already have specs for this endpoint ?

Reply:
Yes, I have added specs for this endpoint.